test/fake_modules/sass/scss2.scss

```diff
@@ -1,7 +1,20 @@
 @use "@test-dep/aFile2";
 @use "mixin"; // should ignore...
 @use "sass:math"; // should show only sass...
+@import "@test-dep/aFile3";
+@forward "@test-dep/aFile4";
+
+%ignore-extend {
+  margin: 0;
+}
 
 div {
+  @extend %ignore-extend;
   font-size: $font-size;
 }
+
+@keyframes ignore-keyframes {
+  70% {
+    margin-left: 90%;
+  }
+}
```

Comment:
I'm not sure I understand why this was added? Is this needed for the tests?

Reply:
yeah I figured it would be useful to validate that `ignore-keyframes` is not treated as an imported dependency. I just gave it some fake properties, lemme know if you'd prefer it empty like `{}` or something

Reply:
Ah! Ok understood, we can leave it then

Thanks!